common/types/encode.go

```diff
@@ -63,6 +63,14 @@ func AtxAsBytes(tx *ActivationTx) ([]byte, error) {
 	return w.Bytes(), nil
 }
 
+func AtxIdsAsBytes(ids []AtxId) ([]byte, error) {
+	var w bytes.Buffer
```

Comment:
we have an InterfaceToBytes method couldn't we just use that ?

Reply:
Fixed.
Now we always use `InterfaceToBytes` unless it is a slice of entities